include/ccf/crypto/jwk.h

```diff
@@ -38,16 +38,12 @@ namespace ccf::crypto
   enum class JsonWebKeyECCurve
   {
     P256 = 0,
-    P256K1 = 1,
-    P384 = 2,
-    P521 = 3
+    P384 = 1,
+    P521 = 2
```

Comment:
This seems risky, but it looks like all serialisation is through JSON, and so may be fine. Worth double checking.

Reply:
Confirmed.